tests/ccl_test_correlation_3d.c

```diff
@@ -165,3 +166,4 @@ CTEST2(corrs_3d,model_3) {
   int model=3;
   compare_correlation_3d(model,data);
 }
+*/
```

Comment:
Why are we removing this test?

Reply:
This test is one of the ones that relies on the now-"forbidden" halofit cosmologies